os/arch/arm/src/amebasmart/Make.defs

```diff
@@ -207,6 +207,7 @@ CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/wifi/driver/inclu
 CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/wifi/driver/intf
 CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/wifi/driver/core
 CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/wifi/api
+CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/wifi/common
```

Comment:
hi Mr. Chang, yes there's one error due to length limitation exceed when building .depend, but it doesn't affect the kernel image generation. We tried some methods but couldn't fix it, could you suggest some ways to fix this issue? Thank You.

Reply:
TizenRT uses few paths for header.
1. os/include
2. framework/include
3. external/include
4. external/include/libcxx

We locate public headers there. 
We can distinguish public and private headers using this.

And we also should use CFLAGS, it's not only for Realtek so that you should not use it fully.

The depend target is necessary in build sequence for several reason and it's failed now due to tooo long CFLAGS.

Reply:
Hi Mr. Chang, I have modified CFLAGS and pushed into this PR. Now the .depend target can be successfully built already. Thank You.